### PR TITLE
Adding certified Seldon Operator

### PR DESCRIPTION
**Pull request by nakfour**
To test this install seldon core as shown in the example kfctl-openshift.yaml
To test please note:
1. Models need to be wrapped with lates python, for my exmaple I used
```
s2i build . seldonio/seldon-core-s2i-python3:0.18 modelfull:0.1
```
2. To get the service serving the model run
```
oc describe sdep <modelname>
```
Run this example model:
```
"apiVersion": "machinelearning.seldon.io/v1alpha2",
    "kind": "SeldonDeployment",
    "metadata": {
        "labels": {
            "app": "seldon"
        },
        "name": "modelfull",
        "namespace": "opendatahub"
    },
    "spec": {
        "annotations": {
            "project_name": "seldon",
            "deployment_version": "0.1"
        },
        "name": "modelfull",
        "oauth_key": "oauth-key",
        "oauth_secret": "oauth-secret",
        "predictors": [
            {
                "componentSpecs": [{
                    "spec": {
                        "containers": [
                            {
                                "image": "nakfour/modelfull:0.1",
                                "imagePullPolicy": "Always",
                                "name": "modelfull",
                                "resources": {
                                    "requests": {
                                        "memory": "10Mi"
                                    }
                                }
                            }
                        ],
                        "terminationGracePeriodSeconds": 20
                    }
                }],
                "graph": {
                    "children": [],
                    "name": "modelfull",
                    "endpoint": {
                        "type" : "REST"
                    },
                    "type": "MODEL"
                },
                "name": "modelfull",
                "replicas": 1,
                "annotations": {
                "predictor_version" : "0.1"
                }
            }
        ]
    }
}

```
```
oc create -f <model.yaml>
```
```
curl -X POST -H 'Content-Type: application/json' -d '{"strData": "0.365194527642578,0.819750231339882,-0.5927999453145171,-0.619484351930421,-2.84752569239798,1.48432160780265,0.499518887687186,72.98"}' http://modelfull-yourclusterserviceurl/api/v1.0/predictions
```

### CODE REVIEW

**Comment by nakfour** [comment]
@crobby this is ready for testing with AI library. Thx

File: odhseldon/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Seldon
+
+Seldon component subscribes to Seldon certified operator, which in turn issues an install of the Seldon certified operator in the namespace. 
+ 
+### Folders
+There are is one main folders in the Seldon component
```
Comment (crobby) [inline odhseldon/README.md:6]: folder instead of folders

File: odhseldon/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+1. cluster: contains the subscription to the Seldon operator
+
+
+
+### Installation
+To install Prometheus add the following to the `kfctl` yaml file.
```
Comment (crobby) [inline odhseldon/README.md:12]: We're not installing Prometheus here.....copy paste, eh?? :)

**Comment by crobby** [comment]
Also, I'd be remiss if I didn't document my call for a basictest for this component

File: odhseldon/README.md (hunk 2 above)
Comment (nakfour) [reply odhseldon/README.md]: lol fixed

File: odhseldon/README.md (hunk 1 above)
Comment (nakfour) [reply odhseldon/README.md:6]: This will stay folders, it is a title of the section listing Folders, in this case it is only one folder

Comment (crobby) [reply odhseldon/README.md:6]: I meant where it says "There are is one main folders".......Should be, "There is one main folder"

**Pull request merged**